nexus/db-queries/src/db/datastore/rack.rs

```diff
@@ -764,6 +765,13 @@ impl DataStore {
                     }
 
                     // Insert the RSS-generated blueprint.
+                    //
+                    // RSS does not run the planner, so we construct an empty
+                    // report for this blueprint.
+                    let blueprint =
+                        BlueprintWithPlanningReport::with_empty_report(
```

Comment:
I wonder if we should have a way to construct a "planning report" that didn't actually come from the planner, and just includes a note about the source? E.g., here, it would be nice if the planning report were able to say "this was produced by RSS" (and similarly for blueprints that are uploaded via `omdb` to Nexus directly).

Doesn't seem particularly impactful either way, but if folks like that idea I'll file an issue for it.